src/azure-cli/azure/cli/command_modules/appconfig/_utils.py

```diff
@@ -254,3 +261,73 @@ def parse_tags_to_dict(tags):
                 tags_dict[tag_key] = tag_value
         return tags_dict
     return tags
+
+
```

Comment:
Should the json code just be it's own file. There is one main entry point with a decent amount of just helper functions.

Reply:
Good call out. This has been updated accordingly